src/SFRField.hpp

```diff
@@ -63,13 +80,24 @@ class SFRField : public SFRInterface
     {
         return value;
     }
-    T get() { return value; }
+
+    T get()
+    {
+        return value;
+    }
+
+    float get_float()
+    {
+        return value / resolution;
```

Comment:
Can we guarantee that value will always have the same precision (when set with input)? If so this all looks good

Reply:
Yes we can guarantee that the same precision is always required